src/app/store/Models/AlbumsModel.ts

```diff
@@ -6,24 +6,42 @@ import uuid from "react-native-uuid"
 export interface Album {
   id: Readonly<string>
   name: string
-  artworkIds?: string[]
-  documentIds?: string[]
+  artworkIds: string[]
+  documentIds: string[]
+  installShotUrls: string[]
   createdAt: string
 }
 
 type ArtistSlug = string
 type ArtworkId = string
 type DocumentId = string
+type InstallShotUrl = string
 export interface AlbumsModel {
   sessionState: {
     selectedArtworksForNewAlbum: Record<ArtistSlug, ArtworkId[]>
     selectedArtworksForExistingAlbum: Record<ArtistSlug, ArtworkId[]>
   }
   albums: Album[]
-  addAlbum: Action<this, { name: string; artworkIds?: ArtworkId[]; documentIds?: DocumentId[] }>
+  addAlbum: Action<
+    this,
+    {
+      name: string
+      artworkIds: ArtworkId[]
```

Comment:
```suggestion
      artworkIds: this["artworkIds"]
```
you could also do this for more futureproof types